services/web/server/src/simcore_service_webserver/director_v2/director_v2_service.py

```diff
@@ -1,4 +1,4 @@
-""" plugin API
+"""plugin API
```

Comment:
internal API = service ?
why keep multiple names?

Reply:
that was old naming...